src/content/ConfirmationModal/ConfirmationModal.css

```diff
@@ -1,13 +1,13 @@
 .ConfirmationModalBackground {
-  background-color: rgba(99, 99, 99, 0.6);
-  position: fixed;
-  top: 0;
-  height: 700px;
-  width: 300px;
-  justify-content: center;
-  align-items: center;
-  display: flex;
-  z-index: 100;
+    background-color: rgba(99, 99, 99, 0.6);
```

Comment:
Some files like this one have weird indentation changes - a `yarn fmt-fix` should take care of them.